test/test_runner.sh

```diff
@@ -1,8 +1,24 @@
 #!/usr/bin/env bash
 set -e
 
+TEST_USERNS=${TEST_USERNS:-}
+
 cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
```

Comment:
Minor: The quoting hurts my eyes.  How about ``cd $(dirname $(readlink -f "$BASH_SOURCE"))``  (imperfect...but more pretty) :grinning: